packages/typespec-azure-resource-manager/src/rules/arm-resource-name-pattern.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  DiagnosticTarget,
+  Program,
+  SourceLocation,
+  createRule,
+  defineCodeFix,
+  getPattern,
+  getSourceLocation,
+} from "@typespec/compiler";
+
+import { isWhiteSpace } from "../../../../core/packages/compiler/src/core/charcode.js";
```

Comment:
hhm this is going to fail in the packaged package

Reply:
I would  just do `=== " "`  think it will work just as well. Or don't reindent and let the formatter do it when they trigger it.